Maple2.File.Parser/TableParser.cs

```diff
@@ -1225,4 +1233,48 @@ public IEnumerable<JobTable> ParseJobTable() {
             yield return (entry.id, entry);
         }
     }
+
+    public IEnumerable<(int Grade, WeddingExp Exp)> ParseWeddingExp() {
+        string xml = Sanitizer.RemoveEmpty(xmlReader.GetString(xmlReader.GetEntry("table/weddingexp.xml")));
+        var reader = XmlReader.Create(new StringReader(xml));
+        var data = weddingExpSerializer.Deserialize(reader) as WeddingExpRoot;
+        Debug.Assert(data != null);
```

Comment:
**suggestion:** Enhance error handling for production scenarios

While Debug.Assert is useful for development, consider implementing more robust error handling for production scenarios, such as throwing a custom exception or logging the error.